### PR TITLE
Update monaco editor package

### PR DESCRIPTION
**Pull request by vivek1729**
- [ x ] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)
- [ x ] I have updated the changelogs/current_changelog.md file with some information about the change that I am making the appropriate file.
- [ x ] I have validated or unit-tested the changes that I have made.
- [ x ] I have run through the TEST_PLAN.md to ensure that my change does not break anything else.

### CODE REVIEW

File: packages/monaco-editor/src/MonacoEditor.tsx [hunk 1]
```diff
@@ -187,14 +187,14 @@ export default class MonacoEditor extends React.Component<IMonacoProps> {
...
-        autoIndent: true,
+        autoIndent: "advanced",
         // Allow editor pop up widgets such as context menus, signature help, hover tips to be able to be
         // displayed outside of the editor. Without this, the pop up widgets can be clipped.
         fixedOverflowWidgets: true,
         find: {
           addExtraSpaceOnTop: false, // pops the editor out of alignment if turned on
           seedSearchStringFromSelection: true, // default is true
-          autoFindInSelection: false // default is false
+          autoFindInSelection: "never" // default is false
```
Comment (jruales) [inline packages/monaco-editor/src/MonacoEditor.tsx]:
```suggestion
          autoFindInSelection: "never" // default is "never"
```

**Review by jruales — APPROVED** [review]

**Comment by jruales** [comment]
@vivek1729 doesn't this change need to be mentioned in a changelog?

File: packages/monaco-editor/src/MonacoEditor.tsx (hunk 1 above)
Comment (vivek1729) [reply packages/monaco-editor/src/MonacoEditor.tsx]: Good catch.

**Comment by vivek1729** [comment]
@captainsafia , do you suggest any updates to the changeLog for this? Also, I am not sure why the dev setup validation is failing. It failed on my local windows machine as well and I worked around it by disabling the file check.

**Review by MSeal — APPROVED** [review]
Doesn't look like any of the breaking changes impact our use / they are already accounted for. The dev build failure is unrelated to the change.

**Comment by captainsafia** [comment]
Thanks folks! I'll work out getting a fresh release of the monaco-editor package with this change.

**Pull request merged**